app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,69 @@
+class RelationshipsController < ApplicationController
+  before_action :find_department, only: %i(new create)
+  before_action :find_user, only: :create
+  before_action :find_relationship, only: %i(update destroy)
+  before_action :paginate_users, only: :new
+  Pagy::DEFAULT[:items] = Settings.relationship.per_page
+
+  def new; end
+
+  def create
+    @user.join_department @department
+    flash[:success] = t ".success_message"
+    redirect_back(fallback_location: root_path)
+  end
+
+  def update
+    role = params[:role]
+    if @relationship.update_role role
```

Comment:
```suggestion
    if @relationship.update_role params[:role]
```
ý chị là ntn
k cần biến role